cfme/intelligence/reports/schedules.py

```diff
@@ -148,33 +160,20 @@ def __str__(self):
     report_filter = attr.ib()
     active = attr.ib(default=None)
     timer = attr.ib(default=None)
-    from_email = attr.ib(default=None)
-    emails = attr.ib(default=None)
+    email = attr.ib(default=None)
     email_options = attr.ib(default=None)
 
-    @property
-    def exists(self):
-        schedules = self.appliance.db.client["miq_schedules"]
-        return self.appliance.db.client.session\
-            .query(schedules.name)\
-            .filter(schedules.name == self.name)\
-            .count() > 0
-
     def update(self, updates):
         view = navigate_to(self, "Edit")
         changed = view.fill(updates)
         if changed:
             view.save_button.click()
+            view.flash.assert_no_error()
         else:
             view.cancel_button.click()
-        view = self.create_view(ScheduleDetailsView, override=updates, wait='10s')
-        view.flash.assert_no_error()
-        if changed:
-            view.flash.assert_message(
-                'Schedule "{}" was saved'.format(updates.get("name", self.name)))
-        else:
-            view.flash.assert_message(
-                'Edit of Schedule "{}" was cancelled by the user'.format(self.name))
+
+        view = self.create_view(ScheduleDetailsView)
```

Comment:
Please use `wait` argument of `create_view()` to trigger an automatic wait_is_displayed call.

Reply:
Why is `override=updates` being removed from this create_view call? If the view's `is_displayed` methods depend on entity attributes that can be changed, then this will never be able to assert the view as it'll be using the original context object.

Reply:
> Please use `wait` argument of `create_view()` to trigger an automatic wait_is_displayed call.

If I add `wait` in the `create_view()`, I wouldn't have anything left to do with that view. I checked other `update()` methods, where the view is used to assert a flash message, but since we have stopped asserting flash messages inside the view, I am not sure what to do with the view that was just created. Suggestions?